src/plugins/kibana_legacy/public/utils/index.ts

```diff
@@ -17,15 +17,12 @@
  * under the License.
  */
 
-import MarkdownIt from 'markdown-it';
-import { uiModules } from 'ui/modules';
-import 'angular-sanitize';
-
-const markdownIt = new MarkdownIt({
-  html: false,
-  linkify: true,
-});
-
-uiModules.get('kibana', ['ngSanitize']).filter('markdown', function($sanitize) {
-  return md => (md ? $sanitize(markdownIt.render(md)) : '');
-});
+export * from './migrate_legacy_query';
```

Comment:
On second thought - is `migrate_legacy_query` more of a `data` plugin responsibility? Not sure how to handle this. Can also be handled in a separate issue

Reply:
about migrate_legacy_query:   
_Creates a standardized query object from old queries that were either strings or pure ES query DSL_

Dear @elastic/kibana-app-arch 🏹  what do you think, should it be migrated to `data`